jwst/ramp_fitting/ramp_fit.py

```diff
@@ -733,13 +901,13 @@ def dq_compress_sect(gdq_sect, pixeldq_sect):
     """
     Extended Summary
     ----------------
-    Get ramp locations where the data has been flagged as saturated in the
-    4D GROUPDQ array for the current data section, find the corresponding image
-    locations, and set the SATURATED flag in those locations in the
-    PIXELDQ array. Similarly, get the ramp locations where the data has been
-    flagged as a jump detection in the 4D GROUPDQ array, find the corresponding
-    image locations, and set the COSMIC_BEFORE flag in those locations in the
-    PIXELDQ array
+    Get ramp locations where the data has been flagged as saturated in the 4D
+    GROUPDQ array for the current data section, find the corresponding image
+    locations, and set the SATURATED flag in those locations in the PIXELDQ
+    array. Similarly, get the ramp locations where the data has been flagged as
+    a jump detection in the 4D GROUPDQ array, find the corresponding image 
+    locations, and set the COSMIC_BEFORE flag in those locations in the PIXELDQ
+    array.
```

Comment:
If saturation and CR flags are copied over from `groupdq` to `pixeldq` doesn't that then mark all of the data for the pixel as bad/unusable? Only the particular groups that are flagged as bad are bad, not all the data for the pixel. Or maybe `pixeldq` is being used in some special way in this function?

Reply:
This pixeldq is not used to flag groups for any computations. It is only used for output in the integration-specific output, and for each ramp has these values:

 0 if all reads in the ramp are good
 2 ("SATURATED") if there are >0 SAT reads and no CRs in the ramp
 4 ("JUMP_DET") if there are >0 CRs and no SATs in the ramp
 6 if there >0 SAT reads and >0 CRs in the ramp

I've updated the comment accordingly.